### DISCUSSION TITLE
What do "Create Styles" and "Update" buttons do?

### DISCUSSION THREAD

**Discussion by nareshbhatia** [six7/figma-tokens#522] [Q&A] [answered]
![CleanShot 2022-03-13 at 03 04 59](https://user-images.githubusercontent.com/1220198/158049170-554d55c4-9c60-45ac-8709-75a013d80435.png)

I assume "Create Styles" generates styles into Figma. But not sure what "Update" does and how it is different from "Import".

**Comment by nareshbhatia** [comment, marked as answer]
After some experimentation, here's my understanding of what is happening:
1. `Import` is **from** Figma **to** Plugin
2. `Create Styles` is **from** Plugin **to** Figma, but only the styles that do not exist in Figma
3. `Update` is again **from** Plugin **to** Figma, but this time only the styles that already exist in Figma, does not create any new ones.

Please let me know if this is correct.

**Reply by six7** [reply]
Right, that is how it works. The plugin allows you to apply design decisions directly on layers, e.g. borderRadius. Update does both styles and layer updates, however those 3 buttons will be reworked to make it easier. Thank you for that feedback 🙏

Import currently only imports styles, create styles does what you described and Update will just update, not create.

**Reply by nareshbhatia** [reply]
Thanks for clarifying, @six7. I am loving this plugin so far. All improvements will be icing on the cake :-)
